tests/acceptance/features/webUIFiles/copyCurrentPath.feature

```diff
@@ -0,0 +1,13 @@
+Feature: copy current path as a permanent link
```

Comment:
is it a private link or a permanent link? see  #2049

Reply:
hm. permanent in the first place as it points to the same file for any user and this is the only reason to copy it

Reply:
file name of the feature could be: `copyPrivateLinks.feature` and in there could be Scenarios of copying private links of the current folder but also of any file/folder from the side panel

Reply:
file has been renamed